test/parallel/test-cli-node-options.js

```diff
@@ -14,6 +14,8 @@ tmpdir.refresh();
 const printA = require.resolve('../fixtures/printA.js');
 expect(`-r ${printA}`, 'A\nB\n');
 expect(`-r ${printA} -r ${printA}`, 'A\nB\n');
+expect(`  -r ${printA} -r ${printA}`, 'A\nB\n');
```

Comment:
Maybe also add some whitespace before the second `-r`?

Reply:
added